src/openms/source/ANALYSIS/ID/ConsensusIDAlgorithmPEPMatrix.cpp

```diff
@@ -40,46 +40,6 @@ namespace OpenMS
 {
   ConsensusIDAlgorithmPEPMatrix::ConsensusIDAlgorithmPEPMatrix()
   {
-    setName("ConsensusIDAlgorithmPEPMatrix"); // DefaultParamHandler
```

Comment:
I'm reasonable sure that you still need this code...

Removing values from 'DefaultParamHandler' has influcences on the TOPP interface of `ConsensusID`.
Compare the INI files created by `ConsensusID -write_ini test.ini` before and after your changes and you will see there are some things missing (at least there should be)...